docs/items/validations.md

```diff
@@ -0,0 +1,14 @@
+---
+id: validations
+title: Validation Requests in CAT 
+sidebar_position: 2
+---
+
+## Validation Requests in CAT 
+<br/>
+<p>A Validation Request is a prerequisite for Identified users to  upgrade to the Validated type of user.
+<br/>Through this procedure users can gain permissions to submit assessments or self-assessments and interact further with CAT Service features.
+<br/>The user submits the Validation Request, assigned with a specific role (Actor) on behalf of a specific Organisation. </p>
+<p>The procedure is completed successfully when an Admin user approves the validation request.</p>
+
+<b>For demonstration purposes, Validation Requests are approved automatically, at the <a href="https://cat.argo.grnet.gr">demo instance</a>  of CAT Service</b>
```

Comment:
In the <a href="https://cat.argo.grnet.gr">DEMO instance of the Compliance Assessment Toolkit , Validation Requests are approved automatically.